modules/ctl/src/main/scala/Parsers.scala

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.ctl
```

Comment:
This is more or less the same commandline parsing code from before but it's better organized.